plugins/webhooks/index.js

```diff
@@ -78,7 +78,8 @@ function startPRJob(options, request) {
                 type: 'pr',
                 workflow: [job.name],
                 username,
-                sha
+                sha,
+                causeMessage: `${options.action} by ${username}`
```

Comment:
I think it will be more clear to add `Pull request` in front of this: `Pull request synchronized by ... `